libcaf_net/caf/net/stream_transport.hpp

```diff
@@ -84,7 +84,9 @@ class stream_transport : public stream_transport_base<Application> {
                     << CAF_ARG(this->handle_.id) << CAF_ARG(*num_bytes));
       this->collected_ += *num_bytes;
       if (this->collected_ >= this->read_threshold_) {
-        if (auto err = this->next_layer_.handle_data(*this, this->read_buf_)) {
+        if (auto err = this->next_layer_.handle_data(*this,
+                                                     make_span(
```

Comment:
You've removed the `as_bytes`, but this should also compile without `make_span`, does it not?

Reply:
It should compile without `make_span` but isn't it clearer like this?

Reply:
I don't think so. 😉

The function takes a span, which is just a view into *some* sort of buffer. Just like we wouldn't manually convert a string to a `string_view`, this only adds extra noise and verbosity.

Reply:
Alright, I get that. I will remove those 👍

Reply:
Done.